contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -9,7 +9,11 @@ ZETAE2E_CMD=$1
 OPTION=$2
 
 echo "waiting for geth RPC to start..."
-sleep 2
+ if [ "$OPTION" != "import-data" ]; then
+    sleep 2
+  else
+    sleep 520
```

Comment:
Why does it need to be that long?

Reply:
I would imagine we could also replace this with a RPC which tests if the import has been completed?

unrelated, but I think we should replace most/all of these shell scripts with go so you can easily make type safe RPCs

Reply:
Yeah in general there are lot of improvement we can make in the sh script in localnet.
I think we could even consider about moving away from shell. Go?

Reply:
> I think we could even consider about moving away from shell. Go?

Yeah I think go is the best choice because you can make type safe RPCs

Reply:
sounds good , i think go is a good option